tests/stubs/with-phpcs/travis.stub

```diff
@@ -12,5 +12,5 @@ before_script:
   - travis_retry composer install --no-interaction
 
 script:
-  - vendor/bin/php-cs-fixer fix --diff --verbose --dry-run
+  - composer cs-lint
```

Comment:
Why are we adding the cs-lint on the stub files? I thought we were just gonna add the `2.0.0-alpha` release just on the construct codebase.